recipes/carbon.rb

```diff
@@ -17,25 +17,29 @@
 # limitations under the License.
 #
 
-python_runtime 'carbons_python' do
-  provider :system
-  version '2.7'
-  options pip_version: true
+include_recipe 'graphite::_user'
```

Comment:
One for the future but we need to look at removing these recipes and replacing them with resources.